docs/create-component.md

```diff
@@ -0,0 +1,72 @@
+# Create Component Script
```

Comment:
We can update this documentation to include React Native once the script has been created for it